security/pkg/pki/ca/ca.go

```diff
@@ -350,3 +597,31 @@ func updateCertInConfigmap(namespace string, client corev1.CoreV1Interface, cert
 	cmc := configmap.NewController(namespace, client)
 	return cmc.InsertCATLSRootCert(certEncoded)
 }
+
+// updateCertInConfigmapWithRetry lets the self-signed Citadel update root cert
```

Comment:
Can you write a similar function into here instead:
https://github.com/istio/istio/blob/master/security/pkg/k8s/configmap/configmap.go

Reply:
This method is moved to configmap/configmap.go now.